index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+<body>
+  <nav class="navbar">
+    <h4 class="name">Oscar Espinoza</h4>
+    <div class="toggle-btn">
+      <span></span>
+      <span></span>
+      <span></span>
+    </div>
+    <div class="nav-links">
+      <p>Portfolio</p>
+      <p>About</p>
+      <p>Contact</p>
+    </div>
+  </nav>
+  <div class="headline">
+    <h1>Hey there. <br>I'm Oscar</h1>
+    <h2>I'm a software developer</h2>
+    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
```

Comment:
- [ ] Good work creating the first section of the mobile with the corresponding order of the `h` and `p` tags ❗❗❗
Please modify this container so it has a `section` tag instead of a `div`, since the section tag it's applying the best practice when you are creating an entry section on a webpage. 💯  

- [ ] Please add a display flex property to the header

This gives us a lot of options to align all our elements, plus, it's responsive 🟢 🟢